.nvmrc

```diff
@@ -1 +1 @@
-16.17.0
+22.15.0
```

Comment:
Updated to current version. Node 23 is known to have issues with SB so I haven't pushed it that far.